### ISSUE TITLE
Can't use it in WSL Fedora

### ISSUE THREAD

**Issue by delucca** [arkane-systems/genie#272]
**Windows version (build number):**
11

**Linux distribution:**
Fedora 35

**Kernel version:**
5.10.102.1-microsoft-standard-WSL2

**Genie version:**
Can't get it, `genie --version` outputs:
```
Traceback (most recent call last):
  File "/home/delucca/.pyenv/versions/3.10.2/lib/python3.10/runpy.py", line 196, in _run_module_as_main
    return _run_code(code, main_globals, None,
  File "/home/delucca/.pyenv/versions/3.10.2/lib/python3.10/runpy.py", line 86, in _run_code
    exec(code, run_globals)
  File "/usr/lib/genie/genie/__main__.py", line 13, in <module>
  File "/usr/lib/genie/genie/nsenter/__init__.py", line 8, in <module>
  File "/home/delucca/.pyenv/versions/3.10.2/lib/python3.10/ctypes/__init__.py", line 8, in <module>
    from _ctypes import Union, Structure, Array
ModuleNotFoundError: No module named '_ctypes'
```

**Describe the bug**
If I try to run any `genie` command I got the following error:
```
Traceback (most recent call last):
  File "/home/delucca/.pyenv/versions/3.10.2/lib/python3.10/runpy.py", line 196, in _run_module_as_main
    return _run_code(code, main_globals, None,
  File "/home/delucca/.pyenv/versions/3.10.2/lib/python3.10/runpy.py", line 86, in _run_code
    exec(code, run_globals)
  File "/usr/lib/genie/genie/__main__.py", line 13, in <module>
  File "/usr/lib/genie/genie/nsenter/__init__.py", line 8, in <module>
  File "/home/delucca/.pyenv/versions/3.10.2/lib/python3.10/ctypes/__init__.py", line 8, in <module>
    from _ctypes import Union, Structure, Array
ModuleNotFoundError: No module named '_ctypes'
```

I already have `libffi-devel` installed. I'm using `pyenv` with Python on version `3.10.2`

**Comment by delucca** [comment]
> Did you have libffi-devel installed before installing python/pyenv? I'm told it can be order-dependent.

I'm not 100% sure. I've installed `pyenv` with `brew`. In any case, I've just uninstalled it (with `brew uninstall pyenv`) and them installed it again (with `brew install pyenv`). I did it in the same shell session.

In any case, the issue persisted.

**Comment by cerebrate** [comment]
Okay, let's see if your python is working outside of genie. Start an interactive python, and enter the following:

```
import ctypes
libc = ctypes.CDLL("libc.so.6")
libc
libc.time
print (libc.time())
```

And send me the complete output from the python header on, please.

**Comment by delucca** [comment]
```
~
❯ python
Python 3.10.2 (main, May 26 2022, 11:30:24) [GCC 11.2.1 20220127 (Red Hat 11.2.1-9)] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> import ctypes
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/home/delucca/.pyenv/versions/3.10.2/lib/python3.10/ctypes/__init__.py", line 8, in <module>
    from _ctypes import Union, Structure, Array
ModuleNotFoundError: No module named '_ctypes'
>>>
```

Yeah, it seems `ctypes` is not installed within my Python interpreter 🤔

**Comment by delucca** [comment]
Seems to be an issue with Pyenv: https://github.com/pyenv/pyenv/issues/2207

Also, the date of that issue matches the date I've installed Pyenv xD

**Comment by delucca** [comment]
I was able to fix it following this tutorial: https://dev.to/ajkerrigan/homebrew-pyenv-ctypes-oh-my-3d9

But now I'm getting an error with `psutil`:
```
~
❯ genie
Traceback (most recent call last):
  File "/home/delucca/.pyenv/versions/3.10.0/lib/python3.10/runpy.py", line 196, in _run_module_as_main
    return _run_code(code, main_globals, None,
  File "/home/delucca/.pyenv/versions/3.10.0/lib/python3.10/runpy.py", line 86, in _run_code
    exec(code, run_globals)
  File "/usr/lib/genie/genie/__main__.py", line 14, in <module>
ModuleNotFoundError: No module named 'psutil'
```

**Comment by cerebrate** [comment]
Hm. That's weird, because the Fedora package should have a dependency on the python3-psutil package, which should install that module without the need for pip. Could you check and see if python3-psutil is installed on your system?

**Comment by delucca** [comment]
> Could you check and see if python3-psutil is installed on your system?

Indeed it is:
```
❯ sudo dnf list installed | rg  python3-psutil
python3-psutil.x86_64                             5.8.0-12.fc35                        @fedora
```

But, as you've mentioned, I've installed `pyenv` using Homebrew. Since `brew` creates a new user I don't know if it is able to fetch packages from within my system (which would also explain why the `libffi-devel` is installed but it was missing the `ctypes` packages as well)

**Comment by AH-Merii** [comment]
> Hm. That's weird, because the Fedora package should have a dependency on the python3-psutil package, which should install that module without the need for pip. Could you check and see if python3-psutil is installed on your system?

Had the same problem on Ubuntu, also fixed it using `pip install psutil`

**Comment by cerebrate** [comment]
`psutil` has been included in genie itself for 2.5 and up, which should fix this issue at the cost of some bloat.
